alternator/executor.hh

```diff
@@ -199,13 +208,16 @@ public:
     future<request_return_type> describe_continuous_backups(client_state& client_state, service_permit permit, rjson::value request);
 
     future<> start();
-    future<> stop() { return make_ready_future<>(); }
+    future<> stop() {
+        // disconnect from the value source, but keep the value unchanged.
```

Comment:
I think, or rather hope, that this change isn't needed. Why did you need to "disconnect"? Is it to solve some destruction-order issues? Shouldn't the config be the LAST thing to destroy, after Alternator and all other front-ends have finished their own destruction?
This code isn't terrible (just one line of code) but it ugly and the fact it's needed might hint we have other bugs we just haven't found yet.

Reply:
> I think, or rather hope, that this change isn't needed. Why did you need to "disconnect"? Is it to solve some destruction-order issues?

i added it just for the sake of symmetry. as i thought, if the constructor connects `updateable_value` to the one in the conffile, we should perform the opposite operation when tearing it down, another option is do this in the dtor.

> Shouldn't the config be the LAST thing to destroy, after Alternator and all other front-ends have finished their own destruction? This code isn't terrible (just one line of code) but it ugly and the fact it's needed might hint we have other bugs we just haven't found yet.

i agree that config is the last one to destroy. so we don't need to worry about dangling reference. 
if you still think we should drop this. i am happy to drop this line.